mobile/library/common/jni/jni_utility.cc

```diff
@@ -384,3 +384,55 @@ std::vector<MatcherData> javaObjectArrayToMatcherData(JNIEnv* env, jobjectArray
   }
   return ret;
 }
+
+#define DEFINE_CALL_METHOD(JAVA_TYPE, JNI_TYPE)                                                    \
+  JNI_TYPE call##JAVA_TYPE##Method(JNIEnv* env, jobject object, jmethodID method_id, ...) {        \
+    va_list args;                                                                                  \
+    va_start(args, method_id);                                                                     \
+    JNI_TYPE result = env->Call##JAVA_TYPE##MethodV(object, method_id, args);                      \
```

Comment:
curious about this and below, why the `V` at the end of the method?  could this code be changed to `env->Call##JAVA_TYPE##Method(...)` without the args initialization?